oio/event/filters/notify.py

```diff
@@ -2,19 +2,27 @@
 from oio.event.evob import Event, EventError
 from oio.event.beanstalk import Beanstalk, BeanstalkError
 from oio.event.filters.base import Filter
+from oio.event.consumer import EventTypes
 
 
 class NotifyFilter(Filter):
     def init(self):
         queue_url = self.conf.get('queue_url', 'tcp://127.0.0.1:11300')
-        self.beanstalk = Beanstalk.from_url(queue_url)
-        self.tube = self.conf.get('tube', 'notif')
-        self.beanstalk.use(self.tube)
+        self.beanstalk_notify = Beanstalk.from_url(queue_url)
+        self.beanstalk_rebuild = Beanstalk.from_url(queue_url)
+        self.tube_notify = self.conf.get('tube', 'notif')
+        self.tube_rebuild = self.conf.get('tube', 'rebuild')
+        self.beanstalk_notify.use(self.tube_notify)
+        self.beanstalk_rebuild.use(self.tube_rebuild)
 
     def process(self, env, cb):
         data = json.dumps(env)
+        event = Event(env)
         try:
-            self.beanstalk.put(data)
+            if event.event_type == EventTypes.CONTENT_BROKEN:
+                self.beanstalk_rebuild.put(data)
+            else:
+                self.beanstalk_notify.put(data)
```

Comment:
These modifications are useless. If notifications are needed at the same time as rebuild, there will be two instances of this class, each one writing in its own tube.